packages/reacord/library/core/reacord-discord-js.ts

```diff
@@ -53,31 +87,95 @@ export class ReacordDiscordJs extends Reacord {
 		})
 	}
 
+	/**
+	 * Sends a message to a channel.
+	 *
+	 * @param {Discord.Channel} target - Discord channel object.
+	 * @param {CreateChannelMessageOptions} [options={}] - Options for the channel
+	 *   message. Default is `{}`
+	 * @param {React.ReactNode} [content] - Initial React node content to render.
```

Comment:
The types already exist in TS syntax, so there's no need to copy them to JSDoc

Also, "Default is `{}`" is probably redundant 😅

Reply:
Will fix that later tomorrow as this weekend I was kind of busy 👀